### PR TITLE
Issue639/dpl 124 add the concept of library type to a request {c m v 4}

### LINKED ISSUE #639: DPL-124 - Add the concept of library type to a request (C=M, V=4)
**User story**
NPG need to be able to identify IsoSeq runs. To that end we need to add the concept of library type

**Who are the primary contacts for this story**
Steve

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] When a sample is imported it can accept library type (not required)
- [x] When a run is exported to the MLWH it will include the library type if it exists

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

### PR DESCRIPTION
**Pull request by JamesGlover**
Closes #639

Changes proposed in this pull request:

* Add library type to flowcell message

Dependent on: https://github.com/sanger/unified_warehouse/pull/314 being released before this.

### CODE REVIEW

File: config/pipelines.yml [hunk 1]
```diff
@@ -102,6 +102,10 @@ default: &default
                 tag_set_name:
                   type: :model
                   value: tag.tag_set_name
+                pipeline_id_lims:
```
Comment (andrewsparkes) [inline config/pipelines.yml:105]: Not convinced by the name 'pipeline_id_lims' for Libary type in the messsage as it suggests that pipeline to library type is 1:1. Can it not be 'library_type'?

**Review by andrewsparkes — APPROVED** [review]
Looks clean to me other than my minor comment about the message field name.

File: config/pipelines.yml (hunk 1 above)
Comment (stevieing) [reply config/pipelines.yml:105]: pipeline_id_lims is the name of the column in the warehouse so we can't change it.

**Review by stevieing — APPROVED** [review]

File: config/pipelines.yml (hunk 1 above)
Comment (andrewsparkes) [reply config/pipelines.yml:105]: Ok

Comment (JamesGlover) [reply config/pipelines.yml:105]: Well the column in the warehouse doesn't exist yet, so could be renamed; but this name was picked for consistency with the iseq_flowcell table and was originally specified by NPG (I can't quite remember why they didn't want library_type).

I have highlighted the column name in an email for Carol S, so there might still be a request to change it.

**Pull request merged**